data/content/about.md

```diff
@@ -1,3 +1,51 @@
-Free and guided workshops in Rust – a systems programming language that runs blazingly fast, prevents segfaults, and guarantees thread safety.
+RustBridge is a workshop focused on getting underrepresented people with
+a background in another programming language to learn Rust and join the
```

Comment:
i would say the current curriculum requires some familiarity with programming before attending. i'd rather set conservative expectations and then grow them as we develop the curriculum more.

after having taught several rustbridges, i dont think we could serve people who are new to programming well. i dont think we shouldn't strive for that, but its just not currently the case.